setup/rover_bringup.md

```diff
@@ -156,7 +156,22 @@ ros2 launch osr_bringup osr_mod_launch.py
 
 This will start up the ROS 2 nodes to run the rover with the changes you made in the previous steps.
 
-Any errors or warnings will be displayed there in case something went wrong. If you're using the Xbox wireless controller,
+Any errors or warnings will be displayed there in case something went wrong.  
+
+### Note
+It's pretty normal for the roboclaws to issue one or two Current Error messages on startup, which look like this:
+  ```
+  [roboclaw_wrapper-1] [ERROR] [timestamp] [roboclaw_wrapper]: Motor controller 128 reported error code 0x20003000 (hex: 0x20003000),
+  [roboclaw_wrapper-1] M1 Current Error
+  [roboclaw_wrapper-1] M2 Current Error
+  ```
+  As long as they are not continuous, they are not a problem.
+
+
+- The ROS1::noetic version of gazebo simulation exists at https://github.com/dongjineee/rover_gazebo.
```

Comment:
why is this line here?  There's a ros2 verison [here](https://github.com/nasa-jpl/osr-rover-code/tree/master/ROS/osr_gazebo)